GhidraDocs/css/bootstrap.min.css

```diff
@@ -0,0 +1,7 @@
+/*!
```

Comment:
We intentionally keep the installation guide as one file to make it easy to distribute outside of the Ghidra installation directory.  I wonder how invasive just pasting this entire file at the top of the installation guide would be...

Reply:
That would be not too bad.